### PR TITLE
add ceteris

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
everything is tested and works

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -155,6 +155,48 @@ let BattleMovedex = {
...
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Spikes", target);
+			target.side.addSideCondition('spikes');
+			let stats = [];
+			for (let stat in source.boosts) {
+				// @ts-ignore
+				if (stat !== 'accuracy' && stat !== 'evasion' && source.boosts[stat] < 6) {
+					stats.push(stat);
+				}
+			}
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:183]: Why not just have an array of valid stats?

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -155,6 +155,48 @@ let BattleMovedex = {
...
+			if (stats.length) {
+				let randomStat = this.sample(stats);
+				let boost = {};
+				boost[randomStat] = 1;
+				this.boost(boost, source);
+			} else {
+				return;
+			}
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: no need to else this

File: mods/seasonal/moves.js (hunk 1 above)
Comment (ghost) [reply mods/seasonal/moves.js:183]: copied the code from starf berry. It works just as well tbh

**Pull request merged**